src/examples/block_stream.rs

```diff
@@ -0,0 +1,41 @@
+mod config;
+use std::num::NonZeroU64;
+use iroha_client::client::Client;
+
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let config = config::get_config();
+    let iroha_client: Client = Client::new(&config)?;
+
+    initiate_block_listener(&iroha_client, 1);
+
+    Ok(())
+}
+// Auxiliary method for a block listener
+// You shall implement it first
+fn non_zero_handler(number: u64) -> NonZeroU64 {
+    NonZeroU64::new(number).map_or_else(|| {
+        println!("The number must be > 0, using default value 1");
+        NonZeroU64::new(1).unwrap()
+    }, |non_zero|non_zero)
+
+}
+/*
+A block listener configuration
+    iroha_client(&Client) - Your iroha client implementation
+    initial_block_number(u65) - The number of a block listener should start from.
+    To get total quantity of blocks, you may use method iroha_client.get_status().
+ */
```

Comment:
```suggestion
}

/// A block listener configuration
/// iroha_client - Your iroha client implementation
/// initial_block_number - The number of a block listener should start from.
/// To get total quantity of blocks, you may use method iroha_client.get_status().
```

Использовать `/* ... */` как-то не принято. И тут особенно еще и док-комменты, так что опять три косых черты

Reply:
Так же не нужно указывать типы в документации. Типы видны пользователям из сигнатуры функции

Reply:
Еще по какой-то причине в Rust принято не столько расписывать по_пунктно, зачем нужен каждый параметр, а скорее вплетать параметры в большое предложение, описывающее поведение функции.

Для примера можно обратить внимание на доку к `std`. Еще если в доке упоминаются какие-то элементы кода, а в данном случае имена параметров, то их обычно заключают в `` ` ``